app/src/forms/form/service.js

```diff
@@ -747,6 +749,33 @@ const service = {
     }
   },
 
+  // Set the filesApiAccess boolean for the api key
+  filesApiKeyAccess: async (formId, filesApiAccess) => {
+    let trx;
+    try {
+      if (typeof filesApiAccess !== 'boolean') {
+        throw new Problem(422, `filesApiAccess must be a boolean`);
```

Comment:
Awesome that you're adding the check to make sure that the value is valid. And love that you're throwing a Problem, some of the other code is doing weird and wild things but throwing something is super clean.

One request, though: can you change this to a 400 instead of a 422? The API is using 422 in a bunch of places that it shouldn't be used, and we've slowly been replacing them with 400. It would be great to get this done before it gets deployed, so we don't have to change it later.

Reply:
Sounds good. It's done!

Reply:
Looks great!